license_test_config.yml

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+directories:
+  recursive:
+    - rero_ils
+    - tests
+    - ui/src/app
+    - ui/scripts
+    - ui/e2e
+  flat:
+    - .
+  exclude:
+    txt:
+      - rero_ils/modules/notifications/templates/email/availability
+      - rero_ils/modules/notifications/templates/email/due_soon
+      - rero_ils/modules/notifications/templates/email/overdue
+      - rero_ils/modules/notifications/templates/email/recall
+    js:
```

Comment:
Can you put a TODO comment to remove this config once we have refactored the UI application?

Reply:
ok